### DISCUSSION TITLE
Cannot access Github Copilot for free even though I am a student with a student developer pack account

### DISCUSSION THREAD

**Discussion by jibby1729** [community/community#66547] [Copilot Conversations]
### Select Topic Area

Bug

### Body

**I've been using copilot for about a year, and it suddenly says this now** 
![image](https://github.com/community/community/assets/67472507/e4d907da-a155-4631-85d5-e8d11494f51c)
**When I try to look up my copilot acces, it takes me to a billing page (below) instead of it being for free**
![Try Copilot](https://github.com/community/community/assets/67472507/645600e9-6483-4f0c-8715-c87596740d56)
**I just renewed my student developer pack as well, and my access still hasn't returned for Copilot**
![Student Account](https://github.com/community/community/assets/67472507/7f910625-eda2-486a-ac87-d58b08f72dc3)
Is anyone else facing this issue? Copilot is still supposed to be free for accounts with the student developer pack, but I'm only being given the option to pay to access it.

**Comment by haoshenwan** [comment]
metoo

**Comment by JonrGull** [comment]
I got an email saying my access was good and will be reviewed on 2023-10-09 but I don't have access anymore.

**Comment by sinne903** [comment]
me too

**Comment by y-ykcir** [comment]
me too

**Comment by singam-sanjay** [comment]
+1

**Comment by 0xriyadh** [comment]
Same

**Comment by OuadieZerhouni** [comment]
same here

**Comment by devashish2024** [comment]
Hm

**Comment by devashish2024** [comment]
Hey there, I'll be trying to help you.

You will get 1 year of GitHub copilot access with Github Student Pack. Once approved, you might have to wait a few days for the perks to process.

After it's approved, like you got it yet, you need to go here https://github.com/github-copilot/signup.
You might see a screen You are invited to use GitHub Copilot for free, meaning you have access to github copilot. You need to just enable it on the link and that's all, then go to your settings, copilot to see if its active now.

> if I resolved your issue, please consider this as answer, ty!

Source: #113944

**Reply by OuadieZerhouni** [reply]
thanks now i have access

**Reply by devashish2024** [reply]
No problem!

**Comment by devashish2024** [comment]
> ### Select Topic Area
> Bug
> 
> ### Body
> **I've been using copilot for about a year, and it suddenly says this now** ![image](https://private-user-images.githubusercontent.com/67472507/266750425-e4d907da-a155-4631-85d5-e8d11494f51c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxODY3OTgsIm5iZiI6MTcxMzE4NjQ5OCwicGF0aCI6Ii82NzQ3MjUwNy8yNjY3NTA0MjUtZTRkOTA3ZGEtYTE1NS00NjMxLTg1ZDUtZThkMTE0OTRmNTFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDEzMDgxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzOTUwYTEwMTVmMzBiYjFkOTAxYTY0MjdiNjg4YmYzOGNlODY2MTBiYzY4NzIyMGU5YTE3NWQ4YmNmZjllZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5mmzJ2T_CNZwl0D66zj78K8Oau1Iten3kXGfjC2Ksns) **When I try to look up my copilot acces, it takes me to a billing page (below) instead of it being for free** ![Try Copilot](https://private-user-images.githubusercontent.com/67472507/266750453-645600e9-6483-4f0c-8715-c87596740d56.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxODY3OTgsIm5iZiI6MTcxMzE4NjQ5OCwicGF0aCI6Ii82NzQ3MjUwNy8yNjY3NTA0NTMtNjQ1NjAwZTktNjQ4My00ZjBjLTg3MTUtYzg3NTk2NzQwZDU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDEzMDgxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ3NDFmMmY0NDYyNmIxOWQzNzFkZDMyYWFkZTQ1OTI2ZmU0MjNjOTc1YzcwZDQ4MjA4MjU2NDFlY2NhODc1ZWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gGt3mmmTK8w-Zpc6MJS3fj7OobJ3H0N3UNd4HZbqyU0) **I just renewed my student developer pack as well, and my access still hasn't returned for Copilot** ![Student Account](https://private-user-images.githubusercontent.com/67472507/266750500-7f910625-eda2-486a-ac87-d58b08f72dc3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxODY3OTgsIm5iZiI6MTcxMzE4NjQ5OCwicGF0aCI6Ii82NzQ3MjUwNy8yNjY3NTA1MDAtN2Y5MTA2MjUtZWRhMi00ODZhLWFjODctZDU4YjA4ZjcyZGMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDEzMDgxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE3NGE3MTY1NDlkNGNiNWM4NDg1YTlhNjNkOWYwNWY5NmYwN2MxZDljN2UzOGY5MTdlYzY3NzM4YjZhZTQxYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4aIiEfj6yYfbuQrXmS_tWdc1bXmKHV8Y609xXAGF3uU) Is anyone else facing this issue? Copilot is still supposed to be free for accounts with the student developer pack, but I'm only being given the option to pay to access it.

I see, you are having **Your benefits will be available soon.** Can you wait a few days? It has to become purple and you will get benefits.

Like mine: 
![image](https://github.com/community/community/assets/83082760/729619af-2892-4a05-9a65-8216281a30ba)

If you get it purple, you should then get the all benefits. You will have to read my message above also after you've got purple approved to get github copilot.
